test/extended/templates/templateservicebroker_security.go

```diff
@@ -68,7 +69,19 @@ var _ = g.Describe("[templates] templateservicebroker security test", func() {
 		brokercli = client.NewClient(&http.Client{Transport: &http.Transport{TLSClientConfig: &tls.Config{InsecureSkipVerify: true}}}, adminClientConfig.Host+templateapi.ServiceBrokerRoot)
 
 		viewuser = createUser(cli, "viewuser", bootstrappolicy.ViewRoleName)
+		//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, we should see about removing
+		// this add-role-to-user, as it should not be required
+		// give test user access to the openshift project's templates ... now needed with provision's SAR for get on templates (previously commented out)
+		err = cli.AsAdmin().Run("policy").Args("add-role-to-user", "view", viewuser.Name, "-n", "openshift").Execute()
+		o.Expect(err).NotTo(o.HaveOccurred())
+
 		edituser = createUser(cli, "edituser", bootstrappolicy.EditRoleName)
+		//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses, we should see about removing
+		// this add-role-to-user, as it should not be required
+		// give test user access to the openshift project's templates ... now needed with provision's SAR for get on templates (previously commented out)
+		err = cli.AsAdmin().Run("policy").Args("add-role-to-user", "edit", edituser.Name, "-n", "openshift").Execute()
```

Comment:
see prior response ... perhaps the TODO comments need to be tweaked, but I think in general, we simply need to *see* ... or at least I do based on my experience to date